### PR TITLE
Samples README update and yolov3 anchors file addition

### PR DESCRIPTION
**Pull request by adisan3**
* added yolov3 anchors file 
* updated README.md for ovtf samples

### CODE REVIEW

**Review by ck-intel — APPROVED** [review]
looks fine

File: openvino_tensorflow/api.cc [hunk 1]
```diff
@@ -28,19 +28,18 @@ void enable() { Enable(); }
 void disable() { Disable(); }
 bool is_enabled() { return IsEnabled(); }
 
-bool check_backend(char* backend) {
+bool check_backend(std::string backend) {
```
Comment (ck-intel) [inline openvino_tensorflow/api.cc:31]: pls revert it back to ctype function argument style

File: examples/common/utils.py [hunk 2]
```diff
@@ -17,6 +17,28 @@
 # Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 # ==============================================================================
+# MIT License
```
Comment (ck-intel) [inline examples/common/utils.py:20]: pls delete this license text for now and revert it to earlier stage, as the source links are already mentioned. pls do it for all the 3 files which has this license text.

File: openvino_tensorflow/api.cc (hunk 1 above)
Comment (adisan3) [reply openvino_tensorflow/api.cc]: Done

File: examples/common/utils.py (hunk 2 above)
Comment (adisan3) [reply examples/common/utils.py]: removed the other license text

**Pull request merged**